src/Storage.cpp

```diff
@@ -190,6 +192,11 @@ static inline int32_t getItemAddr(sto_item_id_t itemId, uint16_t* maxItemSize =
             size = STRUCT_MEMBER_SIZE(sto_data_t, wifiPassword);
             break;
 
+         case STO_ITEM_WEIGHTSETPOINT:
+            addr = offsetof(sto_data_t, brewTimeMs);
```

Comment:
Hier und in der nächsten Zeile sind copy/paste-Fehler: da muss "weightsetpoint" rein

Reply:
Danke dir!